terraform/environments/mlra/snstopic.tf

```diff
@@ -0,0 +1,23 @@
+# SNS topic for monitoring to send alarms to
+resource "aws_sns_topic" "mlra-alerting-topic-nonprod" {
+  name = "MLRA-Alerting-Topic-NonProd"
+}
```

Comment:
Can we move all resources to a module, in the cloudwatch one instead? This would allow future migration projects to pick them all up with the module so less code repeated :) 
This would mean the resource need to output the SNS topic name to be picked up by the PagerDuty module though.